code/services/ExtensibleSearchService.php

```diff
@@ -155,4 +155,37 @@ public function getSuggestions($term, $pageID, $limit = 5, $approved = true) {
 		return null;
 	}
 
+
+	/**
+	 *	Retrieve the most relevant search suggestions.
+	 *
+	 *	@parameter <{SEARCH_TERM}> string
+	 *	@parameter <{EXTENSIBLE_SEARCH_PAGE_ID}> integer
+	 *	@return array
+	 */
+
+	public function getTypeahead($request, $pageID) {
```

Comment:
This isn't consistent with above, as it's actually passing through all GET parameters, rather than the entire request.

```php
$this->service->getTypeahead($request->getVars()
```